Source/Magick.NET/Shared/Profiles/Exif/ExifDefinition.cs

```diff
@@ -0,0 +1,364 @@
+﻿using System.Collections;
+
+namespace ImageMagick
+{
+    /// <summary>
+    /// Class that can be used to provide a definition of ExifValues within an EXIF profile.
+    /// </summary>
+    public class ExifDefinition
```

Comment:
Do we need to have this public?

Reply:
I was thinking that it would be useful to be able to see the definitions externally. The reason behind the change was to make amending and saving Exif values easier; at the moment there is nothing which tells you the data type, you have to know that before you can use Magick.NET, but allowing calling applications to interrogate the Exif definitions will make that easier.